src/Layer/Layer.js

```diff
@@ -86,6 +89,20 @@ class Layer extends THREE.EventDispatcher {
 
         this.defineLayerProperty('frozen', false);
 
+        if (config.zoom != undefined) {
+            if (config.zoom.max != undefined && config.zoom.min != undefined) {
+                this.zoom = { max: config.zoom.max, min: config.zoom.min };
+            } else if (config.zoom.min != undefined) {
+                this.zoom = { max: Infinity, min: config.zoom.min };
+            } else if (config.zoom.max != undefined) {
+                this.zoom = { max: config.zoom.max, min: config.zoom.min };
+            } else {
+                this.zoom = { max: Infinity, min: 0 };
+            }
```

Comment:
What about instead:

```suggestion
            this.zoom = { max: config.zoom.max, min: config.zoom.min || 0 };
            if (this.zoom.max == undefined) {
                this.zoom.max = Infinity;
            }
```

Reply:
yes, just check if `config.zoom` is defined!

Reply:
updated